drivers/SmartThings/zwave-thermostat/src/test/test_qubino_flush_thermostat.lua

```diff
@@ -212,26 +212,6 @@ test.register_message_test(
     }
 )
 
-test.register_message_test(
```

Comment:
This can be removed because the device [profile](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/494974b8c37e1ab3274001b8d43ebe2e7083ca80/drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat.yml#L1) does not support the coolingSetpointCapability. I would assume that the profile is correct, but I am not sure how this device works given this [preference](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/494974b8c37e1ab3274001b8d43ebe2e7083ca80/drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat.yml#L24-L34) that tells the user to re-add the device after changing it. Any ideas on if the test or the profile should change?

Reply:
it uses profile switching I think, but looks like it's using default handling for these attributes anyway